doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -99,14 +99,26 @@ Keywords
   The following character sequences are reserved keywords that cannot be
   used as identifiers::
 
-    _ Axiom CoFixpoint Definition Fixpoint Hypothesis IF Parameter Prop
-    SProp Set Theorem Type Variable as at by cofix discriminated else
-    end exists exists2 fix for forall fun if in lazymatch let match
-    multimatch return then using where with
+    _ Axiom CoFixpoint Definition Fixpoint Hypothesis Parameter Prop
+    SProp Set Theorem Type Variable as at cofix else
+    end fix for forall fun if in let match
+    return then where with
```

Comment:
These appear in the list of identifiers used in the definition of Gallina below.  I think you want to remove them from the list of reserved keywords, right?:

Axiom CoFixpoint Definition Fixpoint Hypothesis Parameter Theorem Variable

Reply:
I'm very sorry. I had done it, then added `Admitted` @ cie in the list then apparently mixed up what I did the interactive rebase.